lib/src/common/model/result_code.dart

```diff
@@ -0,0 +1,16 @@
+enum ResultCode {
+  authenticationFinished,
```

Comment:
Would obfuscation change these values? If so, then comparing the string like here will probably not work https://github.com/Adyen/adyen-flutter/pull/170/files#diff-bf50349d444748b5774562dbf3c9fcca4e2694d4cd5c29d6391ff204ad04203dR44